zipkin-autoconfigure/ui/src/main/java/zipkin2/autoconfigure/ui/ZipkinUiAutoConfiguration.java

```diff
@@ -77,15 +77,6 @@
 @EnableConfigurationProperties(ZipkinUiProperties.class)
 @ConditionalOnProperty(name = "zipkin.ui.enabled", matchIfMissing = true)
 class ZipkinUiAutoConfiguration {
-  static final HttpHeaders CACHE_YEAR =
-    HttpHeaders.of(CACHE_CONTROL, "max-age=" + TimeUnit.DAYS.toSeconds(365));
-  static final HttpHeaders CONFIG_HEADERS = HttpHeaders.of(HttpStatus.OK)
-    .add(HttpHeaders.of(CONTENT_TYPE, "application/json"))
-    .add(HttpHeaders.of(CACHE_CONTROL, "max-age=" + TimeUnit.MINUTES.toSeconds(10)));
-  static final HttpHeaders INDEX_HEADERS = HttpHeaders.of(HttpStatus.OK)
```

Comment:
bug was this `HttpHeaders.of(HttpStatus.OK)` which was a hack no longer needed as cacheControl can be done directly now.